src/test/java/org/folio/marc4ld/mapper/Bibframe2Marc245IT.java

```diff
@@ -123,9 +123,7 @@ private Resource createResourceWithWorkWith245() {
     ).setLabel("Instance Title empty");
 
     var work = MonographTestUtil.createResource(
```

Comment:
I think we no longer need `work` for this test.
Also, line 133 (outgoingResources.put(PredicateDictionary.INSTANTIATES, List.of(work));) can be removed.

Reply:
removed